onpremise/setup/setting-up-redash-instance.md

```diff
@@ -60,11 +60,14 @@ Please note that currently the Google Compute Engine images are for version 0.9.
 
 1. Make sure you have a Docker machine up and running.
 2. Make sure your current working directory is the root of this GitHub repository.
-3. Run `docker-compose up postgres`.
-4. Run `./setup/docker/create_database.sh`. This will access the postgres container and set up the database.
-5. Run `docker-compose up`
-6. Run `docker-machine ls`, take note of the ip for the Docker machine you are using, and open the web browser.
-7. Visit that Docker machine IP at port 80, and you should see a Redash login screen.
+3. Run `cp docker-compose-example.yml docker-compose.yml`.
```

Comment:
Let's change this to "Copy `docker-compose-example.yml` to `docker-compose.yml` and update the configuration values along with the Postgres volume location (if needed)." - more verbose but encourages people to understand what they are doing :) Also they really should change the cookie secret if it's not just for testing purposes.